public/doc/swagger-2.yaml

```diff
@@ -623,6 +623,9 @@ definitions:
       - name
       - description
     properties:
+      id:
+        type: integer
```

Comment:
@syncrou I think all ids should be strings and not numbers, since we could have big ints